metrics/workqueue.go

```diff
@@ -26,11 +26,13 @@ import (
 // WorkqueueProvider implements workqueue.MetricsProvider and may be used with
 // workqueue.SetProvider to have metrics exported to the provided metrics.
 type WorkqueueProvider struct {
-	Adds         *stats.Int64Measure
-	Depth        *stats.Int64Measure
-	Latency      *stats.Float64Measure
-	Retries      *stats.Int64Measure
-	WorkDuration *stats.Float64Measure
+	Adds                           *stats.Int64Measure
```

Comment:
cc @yanweiguo 

This changed dramatically since 1.12 (the deprecated stuff goes away in 1.16)